pkg/cmd/server/bootstrappolicy/securitycontextconstraints.go

```diff
@@ -228,6 +234,38 @@ func GetBootstrapSecurityContextConstraints(sccNameToAdditionalGroups map[string
 			// drops unsafe caps
 			RequiredDropCapabilities: []kapi.Capability{"KILL", "MKNOD", "SYS_CHROOT"},
 		},
+		// SecurityContextConstraintsHostNetwork allows host network and host ports
+		{
+			ObjectMeta: kapi.ObjectMeta{
+				Name: SecurityContextConstraintsHostNetwork,
+				Annotations: map[string]string{
+					DescriptionAnnotation: SecurityContextConstraintsHostNetworkDesc,
+				},
+			},
+			AllowHostNetwork:          true,
+			AllowHostPorts:            true,
+			AllowEmptyDirVolumePlugin: true,
+			SELinuxContext: kapi.SELinuxContextStrategyOptions{
+				// This strategy requires that annotations on the namespace which will be populated
+				// by the admission controller.  If namespaces are not annotated creating the strategy
+				// will fail.
+				Type: kapi.SELinuxStrategyMustRunAs,
+			},
+			RunAsUser: kapi.RunAsUserStrategyOptions{
+				// This strategy requires that annotations on the namespace which will be populated
+				// by the admission controller.  If namespaces are not annotated creating the strategy
+				// will fail.
+				Type: kapi.RunAsUserStrategyMustRunAsRange,
+			},
+			FSGroup: kapi.FSGroupStrategyOptions{
+				Type: kapi.FSGroupStrategyRunAsAny,
```

Comment:
Since our default implementation is not using PVs this could be range based.  Since we have steered clear of that in `restricted` I thought it would make sense to continue it here.  There is a wip (https://github.com/openshift/origin/pull/7334) for enabling it in `restricted` though so I'll make the change